src/dotnet/ToolPackage/ToolCommand.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.ToolPackage
+{
+    internal class ToolCommand
+    {
+        public ToolCommand(string name, FilePath executable)
```

Comment:
It is everything you need to create a shim? What about runner? And the name is vague

Reply:
Currently it is sufficient to create shims since the shim repository is intended only for `dotnet`.  Still, it'd be good to add `Runner` now so we don't have to in the future.

With respect to the name, I agree it is a bit generic.  Any suggestions?

Reply:
This class is tricky. The point of the class is "everything you need for a shim" yet it is in `ToolPackage`.

In the end, it should be a concept to the overall command instead of specific to package or shim. That's why it is called `ToolConfig(should be setting) and Executable` in the beginning. ToolSetting is a concept even the producer need to be aware of, and executable is also a concern both needed. I would rather keep the old tuple. Or just call it toolSettingsAndExecutable. it is better than a vague name, you have to look into the content to understand.

Reply:
I think CommandSettings is good

------------low importance after this line-----------

that why i brought up 7 stage of naming. 

CommandSettingsAndExecutable is honest and complete. to avoid `and` we need to jump to intent. But if we cannot find the right one we go back to the left

http://arlobelshee.com/good-naming-is-a-process-not-a-single-step/
![image](https://user-images.githubusercontent.com/6993335/36172965-5990bd74-10bc-11e8-9498-77b48b526840.png)

Reply:
Renamed to `CommandSettings` and added a `Runner` property in upcoming push.